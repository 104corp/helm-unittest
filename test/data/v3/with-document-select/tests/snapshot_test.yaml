suite: Document Selector snapshot tests
templates:
  - templates/foo-deployment.yaml
  - templates/foo-service.yaml
tests:
  - it: should match snapshot and find document
    documentSelector:
      path: kind
      value: Deployment
    asserts:
      - matchSnapshot: {}
      - hasDocuments:
          count: 1
