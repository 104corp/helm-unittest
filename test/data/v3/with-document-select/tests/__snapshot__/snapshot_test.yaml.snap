should match snapshot and find document:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      name: foo-deployment
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      name: foo-service
